### PR TITLE
avm2: flash.system.Capabilities#playerType should return "Desktop" when the current runtime is AIR runtime(unless the swf is loaded by an HTML page)

### PR DESCRIPTION
**Pull request by lvyitian**
See: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/system/Capabilities.html#playerType

### CODE REVIEW

File: core/src/avm2/globals/flash/system/capabilities.rs [hunk 1]
```diff
@@ -26,7 +27,11 @@ pub fn get_player_type<'gc>(
     let player_type = if cfg!(target_family = "wasm") {
         "PlugIn"
     } else {
-        "StandAlone"
+        //See: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/system/Capabilities.html#playerType
```
Comment (evilpie) [inline core/src/avm2/globals/flash/system/capabilities.rs:30]: It seems like we usually don't really refer to the documentation for specific cases like this.

Comment (lvyitian) [reply core/src/avm2/globals/flash/system/capabilities.rs:30]: For the documentation yes, but there're some places that reference the avmplus source like this, if you think this is unnecessary, you can remove it.

Comment (Lord-McSweeney) [reply core/src/avm2/globals/flash/system/capabilities.rs:30]: I think it should be removed, the documentation is an implicit source for most of these cases.

**Review by Aaron1011 — APPROVED** [review]

**Comment by torokati44** [comment]
Thank you!

**Pull request merged**